<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Padavan_l1_2</class>
 <widget class="QMainWindow" name="Padavan_l1_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>657</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>915</width>
    <height>657</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Level 1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image/resource_rc.qrc">
    <normaloff>:/recurce/icon.png</normaloff>:/recurce/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>915</width>
     <height>657</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>421</width>
        <height>483</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>421</width>
        <height>483</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/recurce/Padavan_l1_2_image.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Line" name="line">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image:url(:/recurce/map.png);
border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/recurce/book.png);
border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/recurce/next_btn.png);
border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/recurce/home.png);
border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>PT Sans</family>
        <pointsize>23</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>PT Sans</family>
           <pointsize>23</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Подземелье&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>445</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>445</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
/* VERTICAL SCROLLBAR */
 QScrollBar:vertical {
	border: none;
    background:rgb(255, 255, 255);
    width: 14px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color:rgb(85,116,226);
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::handle:vertical:hover{	
	background-color:rgb(85,116,226);
}
QScrollBar::handle:vertical:pressed {	
	background-color:rgb(85,116,226);
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color:rgb(255, 255, 255);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color:rgb(255, 255, 255);
}
QScrollBar::sub-line:vertical:pressed {	
	background-color:rgb(255, 255, 255);
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color:rgb(255, 255, 255);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color:rgb(255, 255, 255);
}
QScrollBar::add-line:vertical:pressed {	
	background-color:rgb(85,116,226);
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">
border: none;</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-1105</y>
            <width>407</width>
            <height>1600</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1600</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>22</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Тест 1. Вывод данных&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;1. Какая команда отвечает&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;за вывод   на экран?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="question_1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>180</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
border: none;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QRadioButton" name="q1_ask4">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>System.out.printf()</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q1_ask2">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Write()</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q1_ask1">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>print()</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q1_ask3">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Concole.WriteLine()</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>250</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;2. При помощи какой&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;команды можно осуществить&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;вывод ‘ab’ на экран после&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;выполнения &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;этой программы:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>120</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 20px;
background-color: rgb(71,91,142);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;a = ‘a’&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;b = ‘b’&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="question_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>180</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
border: none;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QRadioButton" name="q2_ask4">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>print(a+b)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q2_ask1">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Write(a+b)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q2_ask3">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>print(a,b)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q2_ask2">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Write(a, b)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>250</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;3. При помощи какой &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;команды можно осуществить&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;вывод суммы переменных&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;на экран после выполнения&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;этой программы:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>90</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 20px;
background-color: rgb(71,91,142);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;a = 1&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;b = 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="question_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>180</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
border: none;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QRadioButton" name="q3_ask4">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>print(a+b)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q3_ask1">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Write(a+b)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q3_ask2">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>print(a,b)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="q3_ask3">
                <property name="font">
                 <font>
                  <family>PT Sans</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Write(a, b)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="proverka">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(71,91,142);
border-radius: 11px;</string>
             </property>
             <property name="text">
              <string>Проверить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../image/resource_rc.qrc"/>
 </resources>
 <connections/>
</ui>
